draft-bbs-signatures.md

```diff
@@ -662,18 +663,23 @@ Parameters:
 
 Definitions:
 
-- expand_message is the corresponding operation associated with the ciphersuite's hash_to_curve_g1 parameter.
-- seed_dst is the octet string representing "BBS-SIG-GENERATOR-SEED" in the ASCII character encoding.
-- hash_output_bytes is the integer given by ceil((log2(r) + k) / 8).
-- generator_dst is the octet string representing "BBS-SIG-GENERATOR-DST" in the ASCII character encoding.
+- expand_message, as defined by the hash-to-curve suite uniquely
+                  identified by the hash_to_curve_id ciphersuite
+                  parameter.
+- seed_dst, is the octet string representing "BBS-SIG-GENERATOR-SEED"
+            in the ASCII character encoding.
+- expand_length, non-negative integer. Defined by the ciphersuite.
```

Comment:
I think this should be in the Parameters section

Reply:
Fixed. Thanks!